Production/crab/config/2016/SM_Higgs.txt

```diff
@@ -7,8 +7,5 @@ ZHToTauTau_M125	                    /ZHToTauTau_M125_13TeV_powheg_pythia8/RunIIS
 WplusHToTauTau_M125	                /WplusHToTauTau_M125_13TeV_powheg_pythia8/RunIISummer16MiniAODv3-PUMoriond17_94X_mcRun2_asymptotic_v3-v2/MINIAODSIM
 WminusHToTauTau_M125            	/WminusHToTauTau_M125_13TeV_powheg_pythia8/RunIISummer16MiniAODv3-PUMoriond17_94X_mcRun2_asymptotic_v3-v2/MINIAODSIM
 VBFHToTauTau_M125	                /VBFHToTauTau_M125_13TeV_powheg_pythia8/RunIISummer16MiniAODv3-PUMoriond17_94X_mcRun2_asymptotic_v3-v2/MINIAODSIM
-VBFHToTauTau_M125-v2	            /VBFHToTauTau_M125_13TeV_powheg_pythia8/RunIISummer16MiniAODv3-PUMoriond17_94X_mcRun2_asymptotic_v3-v2/MINIAODSIM
 GluGluHToTauTau_M125-v2	            /GluGluHToTauTau_M125_13TeV_powheg_pythia8/RunIISummer16MiniAODv3-PUMoriond17_94X_mcRun2_asymptotic_v3-v1/MINIAODSIM
 GluGluHToTauTau_M125-v2         	/GluGluHToTauTau_M125_13TeV_powheg_pythia8/RunIISummer16MiniAODv3-PUMoriond17_94X_mcRun2_asymptotic_v3-v3/MINIAODSIM
```

Comment:
the name (first column) is still the same: each task should have a different name